packages/rke2-calico/package.yaml

```
@@ -1,5 +1,5 @@
-url: https://github.com/projectcalico/calico/releases/download/v3.24.1/tigera-operator-v3.24.1.tgz
-packageVersion: 03
+url: https://github.com/projectcalico/calico/releases/download/v3.24.5/tigera-operator-v3.24.5.tgz
+packageVersion: 04
```

```suggestion
packageVersion: 01
```
When there is a version update the package version counter is restarted.